allennlp/tests/data/tokenizers/word_tokenizer_test.py

```diff
@@ -27,10 +27,54 @@ def test_batch_tokenization(self):
             for batch_word, separate_word in zip(batch_sentence, separate_sentence):
                 assert batch_word.text == separate_word.text
 
-    def test_stems_and_filters_correctly(self):
+    def test_stems_and_filters_stopwords_correctly(self):
         tokenizer = WordTokenizer.from_params(Params({'word_stemmer': {'type': 'porter'},
                                                       'word_filter': {'type': 'stopwords'}}))
         sentence = "this (sentence) has 'crazy' \"punctuation\"."
         expected_tokens = ["sentenc", "ha", "crazi", "punctuat"]
         tokens = [t.text for t in tokenizer.tokenize(sentence)]
         assert tokens == expected_tokens
+
+    def test_stems_and_filters_stopwords_from_file_correctly(self):
```

Comment:
It'd be better to put these tests in a (new) `word_filter_test.py` file, instead of testing the additional functionality indirectly here.